pkg/asset/internal/templates.go

```diff
@@ -1263,6 +1265,8 @@ spec:
               port: 9099
             periodSeconds: 10
           volumeMounts:
+            - mountPath: /lib/modules
+              name: lib-modules
```

Comment:
You should add `readOnly: true` to the lib-modules mount. The Calico manifests at docs.projectcalico.org have it set so it should also be here.

Reply:
added